timer/db.py

```diff
@@ -66,9 +66,9 @@ def get_activities(name: Optional[str]) -> dict[str, float]:
             )
         results = session.exec(statement)
         for row in results:
-            activities[row.name] += row.duration_in_minutes
+            activities[row.name] += row.duration_in_seconds
```

Comment:
this needs to be int, we're converting seconds to minutes in the wrong place, best to do it in the final output (show)